libraries/protocol/include/graphene/protocol/asset_ops.hpp

```diff
@@ -446,10 +446,20 @@ namespace graphene { namespace protocol {
          uint64_t fee = 20 * GRAPHENE_BLOCKCHAIN_PRECISION;
       };
 
+      struct additional_options_type
+      {
+         /// Which asset to claim fees from. This is needed, e.g., to claim collateral-
+         /// denominated fees from a collateral-backed smart asset. If unset, assumed to be same
+         /// asset as amount_to_claim is denominated in, such as would be the case when claiming
+         /// market fees.
+         fc::optional<asset_id_type> claim_from_asset_id;
+      };
+
       asset           fee;
-      account_id_type issuer;
-      asset           amount_to_claim; /// amount_to_claim.asset_id->issuer must == issuer
-      extensions_type extensions;
+      account_id_type issuer; /// must match issuer of asset from which we claim fees
```

Comment:
We need to use `///<` for docs for the field in the same line. See https://doxygen.bitshares.org/structgraphene_1_1protocol_1_1asset__claim__fees__operation.html#aec43fc4728468c7641de361ac578b3db

Reply:
Fixed in 02f9da8.